build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -181,13 +180,7 @@ Future<bool> _allMigratedToNullSafety(PackageGraph packageGraph,
       return false;
     }
 
-    final definedLanguageVersion = parsedFile.unit.languageVersionToken;
-    if (definedLanguageVersion != null) {
-      version = Version(
-          definedLanguageVersion.major, definedLanguageVersion.minor, 0);
-    }
-
-    if (version < _minVersionForNullSafety) return false;
+    return parsedFile.unit.featureSet.isEnabled(Feature.non_nullable);
```

Comment:
I realized I gave you the wrong code here, :)
```suggestion
    if (!parsedFile.unit.featureSet.isEnabled(Feature.non_nullable)) return false;
```